flask_monitoringdashboard/database/stack_line.py

```diff
@@ -32,7 +32,7 @@ def add_stack_line(db_session, request_id, position, indent, duration, code_line
     )
 
 
-def get_profiled_requests(db_session, endpoint_id, offset, per_page):
+def get_profiled_requests(db_session, endpoint_id, sortby, offset, per_page):
```

Comment:
consider adding: `sortby=Request.duration`.
Then the lines 45-48 can be removed

Reply:
Hi,

Thanks for the suggestions, I have implemented the suggested changes. However, I couldn't get it to work this way and worked around it.

Do review the changes and let me know if any other changes are needed.